### PR TITLE
Add support for All Notes Off (CC 123)

### PR DESCRIPTION
**Pull request by jofemodo**

### CODE REVIEW

**Comment by jpcima** [comment]
Seems a `break` has been forgotten in switch

File: raffo.cpp [hunk 1]
```diff
@@ -167,6 +167,10 @@ void RaffoSynth::handle_midi(uint32_t size, unsigned char* data) {
         period = sample_rate * 4 / key2hz(data[1]);
         break;
       }
+      case (0xB0): { // Control Change => All Notes Off (CC 123)
+        if (data[1]!=0x7B) break;
```
Comment (nicoroulet) [inline raffo.cpp:171]: How about 
```
if (data[1] == 0x7B) {
    keys.clear();
}
break;
```

**Comment by nicoroulet** [comment]
Please squash commits for small PRs (also, the first commit and merge commit are the same and have been sent separately as another pull request, so please remove)

**Comment by jofemodo** [comment]
We want the same behaviour as a note-off, so no break ;-)

P.S: Excuse the little mess with the commits/PRs. I edited directly on github ...

**Comment by nicoroulet** [comment]
Oh, I get it. Could you please add a comment explaining?
PS: we'll translate the spanish comments/variables soon :)

**Comment by jofemodo** [comment]
Done!
BTW, the nice spanish comments are OK for me ;-)

**Comment by nicoroulet** [comment]
Thanks!

**Pull request merged**